tests/roles/ovn_adoption/defaults/main.yaml

```diff
@@ -33,3 +33,16 @@ ovn_nic_mapping_patch: |
         ovnController:
           nicMappings:
             baremetal: baremetal
+
+dpa_dir: "../.."
```

Comment:
Why these variables here?

Reply:
these are the default paths to the adoption repo dir as it is on the controller node in regular jobs
alternatively, in ospdo jobs the adoption repo dir  location would be different, and should be specified with this variable.
This is because of a fundamental difference in how the ospdo job is run , 
the zuul "controller" node is executing scripts on the BM titan node , so that dir locations are relative to the bmaremetal host the titan that's has the test env (odp with ospdo and rhoso) on it

Reply:
These variables are not used in adoption role playbooks. I guess dpa_dir and tests_dir is used in zuul job playbooks. So, shouldn't it be defined as zuul variable? 
Any thoughts @holser @jistr

Reply:
So these were introduced by Pini but i can share my view on things:

`dpa_tests_dir` is used for example here https://github.com/openstack-k8s-operators/data-plane-adoption/blob/df159026b647c743e5f55ca75879189419cd04d1/tests/roles/backend_services/tasks/ospdo_backend_services.yaml#L15

And `dpa_dir` isn't used directly but it allows you to just point it to the DPA repo root directory and then `dpa_tests_dir` will get automatically set to the correct path inside the repo.

The default values should work for all existing jobs, but Pini's environment has the DPA repo not on the Ansible controller so he needs more control over how the paths are resolved.